EHR_SM/resources/etls/animal_template.xml

```diff
@@ -8,7 +8,7 @@
             <description>Copy Animal Data to Animal Source</description>
             <!-- For source, update folderPath to the folder path of your EHR folder using dot notation. queryName can be
```

Comment:
The comment here and a few lines lower is slightly out of date with the new placeholder value